hbw/app/javascript/packs/hbw/components/entity_task.js.jsx

```diff
@@ -92,13 +95,15 @@ modulejs.define(
       };
 
       submitForm = (variables) => {
+        const { refresh } = this.context;
         this.props.env.forms.save({
           taskId: this.props.taskId,
           variables,
           token:  this.props.task.form.csrf_token
         })
           .done(() => this.setState({ error: null }))
           .done(() => this.props.trigger('hbw:form-submitted', this.props.task))
+          .done(() => refresh(this.props.taskId))
```

Comment:
I don't think you should refresh task on submit, because this task probably already completed (this is the point of `submit` button). The only thing you maybe can do here — remove this task from the store, but I think in this case better to wait hook from camunda and do nothing here.

Reply:
But you should do it after claiming task on the form: https://github.com/latera/homs/blob/ed173c7137f4955fca1af87144f7d701d42acc6f/hbw/app/javascript/packs/hbw/components/form.js.jsx#L58-L61